packages/inspector/src/types/string.ts

```
@@ -1,11 +1,9 @@
-import type { InspectType, Options } from "../types";
+import type { Options } from "../types";
 import truncate from "../utils/truncate";
 import wrapQuotes from "../utils/wrap-quotes";
 
-const stringEscapeChars = new RegExp(
-    String.raw`['\0-\u001f\u007f-\u009f\u00ad\u0600-\u0604\u070f\u17b4\u17b5` + String.raw`\u200c-\u200f\u2028-\u202f\u2060-\u206f\ufeff\ufff0-\uffff]`,
-    "g",
-);
+// eslint-disable-next-line no-control-regex, sonarjs/no-control-regex, no-misleading-character-class, sonarjs/no-misleading-character-class
+const stringEscapeChars = /['\0-\u001F\u007F-\u009F\u00AD\u0600-\u0604\u070F\u17B4\u17B5\u200C-\u200F\u2028-\u202F\u2060-\u206F\uFEFF\uFFF0-\uFFFF]/g;
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)